tests/test_router_bgp.rb

```diff
@@ -192,15 +224,19 @@ def test_routerbgp_set_get_bestpath
       assert(bgp.bestpath_cost_community_ignore, "vrf #{vrf}: "\
              'bgp bestpath_cost_community_ignore should be enabled')
       bgp.bestpath_med_confed = true
-      assert(bgp.bestpath_med_confed, "vrf #{vrf}: "\
-             'bgp bestpath_med_confed should be enabled')
+      if node.client.api == 'NXAPI' ||
```

Comment:
These checks are added because only some of the bestpath options are working yet in the node utils for XR, not because of any inherent NX-exclusivity of these options, right? If so, can you please mark these with 'TODO:'?